src/main/java/com/rakovets/course/java/core/practice/strings/StringUtil.java

```diff
@@ -0,0 +1,157 @@
+package com.rakovets.course.java.core.practice.strings;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+public class StringUtil {
+
+    static String connectTwoStrings(String str1, String str2) {
+        return str1 + str2;
+    }
```

Comment:
```suggestion
    public String connectTwoStrings(String str1, String str2) {
        return str1 + str2;
    }
```

This isn't OOP practice